fastlane/Fastfile

```diff
@@ -251,10 +251,17 @@ end
 def setCIEnvironmentVariable(testPlanFile)
```

Comment:
(Sorry for not catching this before!)
This function should be called in the `before_all` lane, `if is_ci` check

Reply:
Yes! 👍

Reply:
Hmmm, we'd have to also adjust the release lane. The function mutates the test plan files, and in the release lane we do:
```python
  sh("git add -A")
  sh("git commit -m 'Bump #{version_number}'")
  sh("git tag #{version_number}")
```
which would commit the changes to the test plan files, too. So, maybe let's keep it and create a separate "Fastfile cleanup" task?

Reply:
Setting up the CI variable locally doesn't do any harm. It basically just makes the timeouts longer and formats the error messages differently, that's it :)

Reply:
-> CIS-223